arch/arc/core/fault.c

```diff
@@ -29,7 +29,7 @@
  *
  * @return This function does not return.
```

Comment:
Please update the 'this function does not return' comment, the function could return for stack canarie exceptions, right?